README.md

```diff
@@ -1,64 +1,44 @@
+ <a href="https://www.infoblox.com">
+    <img src="https://avatars.githubusercontent.com/u/8064882?s=400&u=3b245589302c409aff2ce2ba26d95e6df6cfe342&v=4" alt="Infoblox logo" title="Infoblox" align="right" height="50" />
+</a> 
+ 
 # Terraform Provider for Infoblox
- <img width="171" alt="capture" src="https://user-images.githubusercontent.com/36291746/39614422-6b653088-4f8d-11e8-83fd-05b18ca974a2.PNG">
+Terraform provider plugin to integrate with Infoblox Network Identity Operating System [NIOS].
+The plugin enables lifecycle management of Infoblox NIOS DDI resources.
 
+The latest version of Infoblox NIOS provider is [v1.1.1](https://github.com/infobloxopen/terraform-provider-infoblox/releases/tag/v1.1.1)
+The features in development are available at [`develop`](https://github.com/infobloxopen/terraform-provider-infoblox/tree/develop) branch.
 
-## Build Status
-| Master                                                                                                                                                               | Develop                                                                                                                                                              |
-| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [![Build Status](https://travis-ci.org/infobloxopen/terraform-provider-infoblox.svg?branch=master)](https://travis-ci.org/infobloxopen/terraform-provider-infoblox)  | [![Build Status](https://travis-ci.org/infobloxopen/terraform-provider-infoblox.svg?branch=develop)](https://travis-ci.org/infobloxopen/terraform-provider-infoblox) |
-
-## Requirements
-* [Terraform](https://www.terraform.io/downloads.html) 0.11.x or greater
-* [Go](https://golang.org/doc/install) 1.12.x (to build the provider plugin)
-* CNA License need to be installed on NIOS. If CNA is not installed then following default EA's should be added in NIOS side:
-   * VM Name :: String Type
-   * VM ID :: String Type
+## NIOS Requirements
+CNA License need to be installed on NIOS. If CNA is not installed then following default EA's should be added in NIOS side manually:
```

Comment:
Change EA's to EAs

Reply:
A CNA License needs to be installed on NIOS. If CNA license is not installed then the following default EAs must be added at the NIOS' side manually: